indigo_app/notifications.py

```diff
@@ -95,6 +98,20 @@ def send_templated_email(self, template_name, recipient_list, context, **kwargs)
             fail_silently=settings.INDIGO.get('EMAIL_FAIL_SILENTLY'),
             **kwargs)
 
+    def notify_admins_user_signed_up(self, user):
+        """ Function to send emails to admins to notify them when a new user 
+        signs up
+        """
+        subject = "New User Alert"
+        message = """Hey there. \n\nWe are just writing to let you know that a new user has signed up on {0} with the following details:
```

Comment:
@longhotsummer I've introduced the new-line tags at this point to change the email from this:

![image](https://user-images.githubusercontent.com/8082197/62949387-c2d33b00-bdee-11e9-81eb-ef53da98557b.png)

to this:

![image](https://user-images.githubusercontent.com/8082197/62949407-cc5ca300-bdee-11e9-8882-101363d34cbe.png)

The indentation was not consistent.